### DISCUSSION TITLE
Launching gui_based_control vs. launch_ctrk_interface? Or are these not mutually exclusive?

### DISCUSSION THREAD

**Discussion by sunshineinsandiego** [WPI-AIM/ambf#208] [Q&A]
Hi - Following the instructions [here](https://github.com/collaborative-robotics/surgical_robotics_challenge/tree/master/scripts) to try to launch GUI control of the PSM arms / ECM, but not understanding how exactly to do this.

(1) It is written: 

"Simply run the `launch_crtk_interface.py` script. This script will consequently use the `psm_arm.py`, `ecm_arm.py` and `scene.py` scripts and create ROS topics to publish and receive commands."

-- My belief is that I should run `$python3 launch_crtk_interface.py` in a 3rd terminal with my other two terminals running `$ roscore` and `$ ./ambf_simulator --launch_file <surgical_robotics_challenge>/launch.yaml -l 0,1,3,4,14,15 -p 120 -t 1 --override_max_comm_freq 120`.

(2) Then I should run `$ gui_based_control.py` in a 4th terminal to access the GUIs that will interface with the ROS topics launched in by the `launch_crtk_interface.py` script?

OR are these two separate concepts: the `launch_crtk_interface.py` should be used with ROS messages while the `gui_based_control.py` already includes the interface with the simulated PSMs/ECM?

Thanks!

**Comment by adnanmunawar** [comment]
Can you please ask this in the surgical_robotics_challenge repo :)

**Reply by sunshineinsandiego** [reply]
Of course, my apologies!
